File: QuestoesBeecrowd-Iniciante/1036.c
#include <stdio.h>
#include <stdlib.h>
#include <locale.h>
#include <math.h>

int main()
{
    setlocale(LC_ALL, "pt-br");
    double a, b, c, x1, x2;
    double delta;
    scanf("%lf %lf %lf", &a, &b, &c);
    delta = (b * b) - 4 * a * c;

    if (delta >= 0 && a != 0)
    {
        x1 = (-b + sqrt(delta)) / (2 * a);

        x2 = (-b - sqrt(delta)) / (2 * a);
        printf("R1 = %.5lf\n", x1);
        printf("R2 = %.5lf\n", x2);
    }
    else
    {
        printf("Impossivel calcular\n");
    }

    return 0;
}

File: src/Iniciante/1036.java
package Iniciante;

import java.util.Scanner;
import java.util.Locale;


public class Main {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner input = new Scanner(System.in);
        double a,b,c,x1,x2;
        double delta;
        a = input.nextDouble();
        b = input.nextDouble();
        c = input.nextDouble();
        delta = Math.pow(b, 2)-4*a*c;

        if (delta>=0 && a!=0) {
            x1=(-b+Math.sqrt(delta))/(2*a);

            x2=(-b-Math.sqrt(delta))/(2*a);
            
            String r1 = String.format("%.5f", x1);
            String r2 = String.format("%.5f", x2);
            System.out.println("R1 = "+r1);
            System.out.println("R2 = "+r2);
        }
        else{
            System.out.println("Impossivel calcular");
        }
        input.close();
        
    }
}
